api/test/unit/validation/validationMiddleware.test.js

```diff
@@ -365,4 +365,209 @@ describe('Validate Policy JSON Schema structure', function() {
       done();
     });
   });
+  it('should successfully validate policy schema without start_date and end_date in recurring_schedule', function(done) {
```

Comment:
@fraenkel If you have the "schedules" tag specified, then you've to specify at least one of either specific_date or recurring schedule. Otherwise, why would you have the schedules section at all in the policy?

Can you please point me to one of the tests where your "expect"ations weren't met?